servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/H2ClientParentChannelInitializer.java

```diff
@@ -48,6 +48,8 @@ public void init(final Channel channel) {
                 // The max concurrent streams is made available via a publisher and may be consumed asynchronously
                 // (e.g. when offloading is enabled), so we manually control the SETTINGS ACK frames.
                 .autoAckSettingsFrame(false)
+                // We ack PING frames in KeepAliveManager#pingReceived.
+                .autoAckPingFrame(false)
```

Comment:
All PING frames are handled here and always ack'ed:
https://github.com/apple/servicetalk/blob/6b4a2407b6952e0242b1129a728b8146c015efd8/servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/KeepAliveManager.java#L142-L159